Documentation/debugging/debugging-packages.md

```diff
@@ -4,9 +4,9 @@ Debugging CoreFX build issues
 ## MSBuild debug options
 
 * Enable MSBuild diagnostics log (msbuild.log):
-`msbuild my.csproj /flp:v=diag /t:rebuild`
+`dotnet msbuild my.csproj /flp:v=diag /t:rebuild`
 * Generate a flat project file (out.pp):
-`msbuild my.csproj /pp:out.pp`
+`dotnet msbuild my.csproj /pp:out.pp`
```

Comment:
While we are in here, I think it would make sense to also list `/bl` and http://msbuildlog.com/.  This is pretty much exclusively how I debug MSBuild now days.

Reply:
nice idea, that's definitely worth it!